spring-cloud-skipper/spring-cloud-skipper-shell/pom.xml

```diff
@@ -27,6 +27,11 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<scope>test</scope>
+		</dependency>
 		<dependency>
```

Comment:
Currently fails in build with ` spring-cloud-skipper-shell: No tests to run`. 
We may need to add this temporarily :

```
				<configuration>
					<skipTests>true</skipTests>
				</configuration>
```

Reply:
failNoTests=false should to the job